models/articles-models.js

```diff
@@ -11,4 +11,16 @@ exports.findArticleById = (id) => {
       
       return rows;
     });
-};
\ No newline at end of file
+};
+
+exports.readArticles = () => {
+    return db.query(`SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.article_id) AS comment_count FROM articles
+    LEFT JOIN comments
+    ON articles.article_id = comments.article_id
+    GROUP BY articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url
+    ORDER BY articles.created_at;`).then(({rows}) =>{
```

Comment:
You'll just need to add one more word in this query to get the descending order 😄